sdk/helper/keysutil/policy.go

```diff
@@ -1186,10 +1219,10 @@ func (p *Policy) VerifySignature(context, input []byte, hashAlgorithm HashType,
 		Marshaling:    marshaling,
 		SaltLength:    rsa.PSSSaltLengthAuto,
 		SigAlgorithm:  sigAlgorithm,
-	})
+	}, nil)
 }
 
-func (p *Policy) VerifySignatureWithOptions(context, input []byte, sig string, options *SigningOptions) (bool, error) {
+func (p *Policy) VerifySignatureWithOptions(context, input []byte, sig string, options *SigningOptions, managedKeyParams *ManagedKeyParameters) (bool, error) {
```

Comment:
I think I'd like the policy engine's functions to be more agnostic of managed keys.  Can we put the managed key parameters in the SigningOptions?